Simplify the golang sample to use a hello-world app (#54)
okteto/samples: golang/README.md

```console
okteto> go run main.go
Starting hello-world server...
```

You can now access the Golang Sample App at http://localhost:8080.

## Step 4: Develop directly in the cloud

> Now things get more exciting. Edit the file `main.go` and switch the word `cluster` by `Okteto Cloud` at line 24. Save your changes.

Reviewer: ```suggestion
Now things get more exciting. Edit the file `main.go` and replace the word `cluster` with `Okteto Cloud` on line 24. Save your changes.
```
